src/biokbase/narrative/tests/data/response_data.json

```diff
@@ -401,7 +401,7 @@
                 "widget_info": {
                     "name": "no-display",
                     "params": {
-                        "obj_ref": "18836/5/1",
+                        "obj_ref": "62425/2/12",
```

Comment:
Same thing as above.  The mocked upas were bleeding over into the returned test data.